proto/pulumi/errors.proto

```diff
@@ -23,3 +23,16 @@ message ErrorCause {
     string stackTrace = 2;
 }
 
+// An error that can be returned from a component provider and includes details of the
+// error, which can be multiple properties.
+message InvalidInputPropertiesError {
+    // A single invalid input property.
+    message InvalidInputPropertyError {
```

Comment:
```suggestion
    message PropertyError {
```

You don't want the parent message name in the sub message it leads to really long Go names like pulumirpc.InvalidInputPropertiesError_InvalidInputPropertyError

This would be fine as pulumirpc.InvalidInputPropertiesError_PropertyError